cost/flexera/cco/scheduled_reports/scheduled_report.pt

```diff
@@ -95,6 +104,43 @@ parameter "param_billing_centers" do
   default []
 end
 
+parameter "param_percent_change_threshold" do
+  type "number"
+  category "Policy Settings"
+  label "Percent Change Threshold Value"
```

Comment:
Might be worth adding (%) to the end of label, similar to other policies, to communicate to the user very clearly that they are entering a percentage.